tools/ci/clone_with_retries.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ "$#" -lt 1 ]; then
+    print "usage: $0 [options] <repository to clone>"
+    print "The script will forward all options to git clone"
+    exit 1
+fi
+
+for _ in $(seq 1 5); do
+    git clone "$@" && exit 0;
```

Comment:
What do you think about adding an exponential sleep time before each retry?

Reply:
Oh my yes, I'm a bit embarassed it's not already there :sweat_smile: